calculator.py

```diff
@@ -1,11 +1,47 @@
 def addition(number_1, number_2):
+    return (number_1+number_2)
 
 def subtraction(number_1, number_2):
+    return (number_1-number_2)
 
 def multiplication(number_1, number_2):
+    return (number_1*number_2)
 
 def division(number_1, number_2):
+    return(number_1/number_2)
 
-def get_input():
 
-input()
\ No newline at end of file
+#select option
+
+print("***SIMPLE CALCULATOR***")
+print("Select Operation \n 1.Addition \n 2.Subtraction \n 3.Multiplication \n 4.Division")
+
+opt=int(input("operation : "))
+
+Number1=int(input("Enter the First Number = "))
```

Comment:
use snake_case, camel_case for variable name.
use Double_camel_case only for class names.

Number1 -> number_1
Number2 -> number_2